fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflowSpec.groovy

```diff
@@ -206,19 +183,12 @@ class DruidWorkflowSpec extends Specification {
         def defaultHandler = select.handlerSelector as DefaultWebServiceHandlerSelector
 
         when:
-        handlers = getHandlerChain(defaultHandler.uiWebServiceHandler.next)
+        handlers = getHandlerChain(defaultHandler.webServiceHandler.next)
 
         then:
         [AsyncWebServiceRequestHandler, DebugRequestHandler].every {
             handlers.find(byClass(it)) != null
         }
-        when:
-        handlers = getHandlerChain(defaultHandler.nonUiWebServiceHandler.next)
-
-        then:
-        [AsyncWebServiceRequestHandler, DebugRequestHandler, WeightCheckRequestHandler].every {
```

Comment:
Shouldn't we move `WeightCheckRequestHandler` up to the previous `then` section so that we're still testing that it's in the workflow?